components/service/nimbus/build.gradle

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdkVersion config.compileSdkVersion
+
+    defaultConfig {
+        minSdkVersion config.minSdkVersion
+        targetSdkVersion config.targetSdkVersion
+    }
+
+    buildTypes {
+        debug {
+            // Export experiments proguard rules even in debug since consuming apps may still
+            // enable proguard/R8
+            consumerProguardFiles 'proguard-rules-consumer.pro'
+        }
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            consumerProguardFiles 'proguard-rules-consumer.pro'
+        }
+    }
+}
+
+dependencies {
+    api Dependencies.mozilla_nimbus
+
+    implementation Dependencies.kotlin_stdlib
+    implementation Dependencies.kotlin_coroutines
+    implementation project(':support-base')
+    implementation project(':support-locale')
+
+    compileOnly project(":service-glean")
```

Comment:
Right now we don't have a use case for using Nimbus without Glean, and so we have elected to run with the tight linkage for now. Another consideration is that very soon Nimbus' Glean integration will be happening at the Rust layer rather than in Kotlin (once Glean's Rust language bindings are available, and Glean joins the appservices megazord) so this problem of coupling will be the responsibility of the nimbus-sdk.